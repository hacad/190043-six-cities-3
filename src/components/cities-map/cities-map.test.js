import React from "react";
import renderer from "react-test-renderer";
import CitiesMap from "./cities-map.jsx";

jest.mock(`leaflet`);

it(`CitiesMap renders correctly`, () => {
  const city = {name: ``, location: {latitude: 52.38333, longitude: 4.9}};
  const offers = [
    {
      latitude: 52.3909553943508,
      longitude: 4.85309666406198
    },
    {
      latitude: 52.369553943508,
      longitude: 4.85309666406198
    },
    {
      latitude: 52.3909553943508,
      longitude: 4.929309666406198
    },
    {
      latitude: 52.3809553943508,
      longitude: 4.939309666406198
    }
  ];

  const placesMap = renderer.create(
      <CitiesMap city={city} offers={offers}/>
  );

  expect(placesMap).toMatchSnapshot();
});
