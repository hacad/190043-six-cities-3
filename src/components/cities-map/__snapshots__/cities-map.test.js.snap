// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CitiesMap renders correctly 1`] = `
<section
  className="cities__map map"
  id="map"
/>
`;
